test/e2e/framework.go

```diff
@@ -1651,10 +1642,25 @@ func randSeq(n int) string {
 	return string(b)
 }
 
+func randSequence(n int) string {
```

Comment:
what's the problem with `randSeq`?

Reply:
It uses alphanumeric characters but namespaces can only be alphabets so I thought of creating two different functions. 
Alternatively I can add two parameters in the signature of randSeq function one for n and other for the character array to be used.

Reply:
namespace can have alphanumeric chars. I think the naming rule for all resources are same.